### PR TITLE
Exercise 01: Rock, Paper, Scissors

### PR DESCRIPTION
**Pull request by caseybaggz**
## Summary
This PR adds the first result to Exercise 01 of creating a Rock, Paper, Scissors game.

#### Types
* Int, String, None, List, Dictionary, Function
#### Other
* Loops
* Conditionals
#### Modules
* Importing and using Py Standard Lib

### CODE REVIEW

File: src/exercise/01/01.py [hunk 1]
```diff
@@ -0,0 +1,60 @@
...
+def determine_win(players, choices):
+  user = players.get('user')
+  ai = players.get('ai')
+
+  if user not in choices:
+    print('You must choose either 1, 2, or 3 as an option.')
+    return
+
+  if ai == user:
+    print('Tie!')
```
Comment (frenetic) [inline src/exercise/01/01.py:45]: Im not sure `ai == user` will ever happen, since `ai` is a dictionary and `user` is a string.

**Review by frenetic — CHANGES_REQUESTED** [review]

File: src/exercise/01/01.py (hunk 1 above)
Comment (caseybaggz) [reply src/exercise/01/01.py:45]: Huh...that actually has worked every time? Did you try running and playing the game?

Comment (caseybaggz) [reply src/exercise/01/01.py:45]: Wait...they are both strings! ai is a random choice of the `id` field of each option.

Comment (frenetic) [reply src/exercise/01/01.py:45]: yeah, my eyes were tricking me.
Looks good!

It is great you are having fun!

**Review by frenetic — APPROVED** [review]

**Pull request merged**